README.md

```diff
@@ -1,3 +1,8 @@
+Eclipse SmartHome now hosts the Orange implementation of a standard OSGi EnOcean base driver.
+
+Orange implementation of OSGi standard EnOcean Device service specification is now available and maintained on eclipse SmartHome git repository under eclipse public licence (https://www.eclipse.org/legal/epl-v10.html). The specification is officially a chapter of OSGi Release 6 Residential specification (http://www.osgi.org/Release6/HomePage). The implementation of an OSGi Device Service specification is called an OSGi base driver. A base driver simplifies the development of applications controlling devices of a specific protocol, here the EnOcean standard radio technology (https://www.enocean-alliance.org).

```

Comment:
I would remove the first sentence here as it is a repetition from the sentence above now.